charts/simple-app/README.md

```diff
@@ -2,7 +2,7 @@
 
 Default Microservice Helm Chart
 
-![Version: 0.17.2](https://img.shields.io/badge/Version-0.17.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
+![Version: 0.18.0](https://img.shields.io/badge/Version-0.18.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
```

Comment:
Edit the `README.md.gotmpl` and add in two sections..

1. An "upgade notes" sectin like https://github.com/Nextdoor/k8s-charts/blob/main/charts/simple-app/README.md.gotmpl#L15 that discusses what we're adding at a high level.

2. A section that discusses exactly how to use the settings...